design/FY2022/enhacement_to_resilience_metric.md

```diff
@@ -0,0 +1,191 @@
+
+|	Hours of Safety [h]	| End Time of the Safety Duration | Safe Temperature Exceedance Hours [h] | Safe Temperature Exceedance OccupantHours [h]
+|-----------------------|---------------------------------|---------------------------------------|------------------|
+|Space<sub>1</sub>||||
+|...||||
+|Space<sub>N</sub>||||
+|Min||||
+|Max||||
+|Average||||
+|Sum||||
+<p style="text-align: center;"> Table 2. Sample report table of hours of safety for hot events.</p>
+
+**Unmet Degree Hours**
+
+|	Cooling Setpoint Unmet Degree Hours [°C•h] | Cooling Setpoint Unmet Degree OccupantHours [°C•h] | Heating Setpoint Unmet Degree Hours [°C•h] | Heating Setpoint Unmet Degree OccupantHours [°C•h]
+|-----------------------|---------------------------------|---------------------------------------|------------------|
+|Space<sub>1</sub>||||
+|...||||
+|Space<sub>N</sub>||||
+|Min||||
+|Max||||
+|Average||||
+|Sum||||
+<p style="text-align: center;"> Table 3. Sample report table of unmet degree hours (UDH).</p>
+
+**Discomfort-weighted Exceedance Hours**
+
+|	Very-cold Exceedance OccupantHours [h] | Cool Exceedance OccupantHours [h] | Warm Exceedance OccupantHours [h] | Very-hot Exceedance OccupantHours [h]
+|-----------------------|---------------------------------|---------------------------------------|------------------|
+|Space<sub>1</sub>||||
+|...||||
+|Space<sub>N</sub>||||
+|Min||||
+|Max||||
+|Average||||
+|Sum||||
+<p style="text-align: center;"> Table 4. Sample report table of discomfort-weighted exceedance hours.</p>
+
+When *ThermalResilienceSummary* is declared in *Output:Table:SummaryReports*, the above three tables will be generated and presented in the tabular reports. In particular, the *Hours of Safety* table will only be presented if an *Output:Table:SummaryReports:ReportPeriod* object is defined for ThermalResilienceSummary report. 
+
+## Testing/Validation/Data Source(s): ##
+
+Two example files (the DOE reference small office and the one zone uncontrolled model) will be modified to demonstrate the use of the new feature. Simulation results will be manually checked/benchmarked using an excel spreadsheet with input and output from EnergyPlus runs.
+
+## IDD Object changes: ##
+
+Two extra fields will be added to the current People object to define the safe temperature thresholds for a certain kind of population.
+
+    People,
+      N7, \field Cold Stress Temperature Threshold
+          \type real
+          \units C
+          \note this is the indoor safe temperature threshold for cold stress
+    \default 15.56
+      N8, \field Heat Stress Temperature Threshold
+          \type real
+          \units C
+          \note this is the indoor safe temperature threshold for heat stress
+          \default 30
+
+We will add a new object *Output:Table:SummaryReports:ReportPeriod* to add supplementary information to define the reporting period of certain summary tables.
+
+    Output:Table:SummaryReports:ReportPeriod,
```

Comment:
I would suggest `OutputControl:Table:ReportPeriod`, similar to `Output:Table:Style`.

Reply:
The intent of the "SummaryReports" indicates the reporting period is only for the tabular reports, but I agree we should follow the "Output:Table:Style". I will change "Output:Table:SummaryReports:ReportPeriod" to "OutputControl:Table:ReportPeriod" in the coming push.